cityofaustin/atd-moped
# Adds council districts to seed data

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/11531

## Testing
**URL to test:** Local

**Steps to test:**
1. Start the hasura cluster
2. Connect to the DB and query the districts. Verify there are 10 districts.

```sql
select * from layer_council_district;
```


---
#### Ship list
